variants/aws-dev/Cargo.toml

```diff
@@ -28,3 +28,21 @@ included-packages = [
 
 [lib]
 path = "lib.rs"
+
+[build-dependencies]
+# core
+os = { path = "../../packages/os" }
```

Comment:
It's weird that `os` shows up in this list now. Isn't it pulled in by the dependency on `release`?